src/Components/Events/GoogleCalendar/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { MainContainer } from '../style';
+
+function GoogleCalendar() {
+  return (
+    <MainContainer>
+        {/* Replace this iframe with the iframe of the google calendar associated with the google account according to which you want the schedule */}
+        <iframe src="https://calendar.google.com/calendar/embed?src=er3gi62a5hip5jnk1f7r0al62s%40group.calendar.google.com&ctz=America%2FLos_Angeles" 
```

Comment:
```suggestion
        <iframe src="https://calendar.google.com/calendar/embed?src=er3gi62a5hip5jnk1f7r0al62s%40group.calendar.google.com" 
```
I think if we don't put the `ctz` query parameter, the  timezone assumed will be the one from the computer accessing the page.